src/applications/mhv-medical-records/tests/containers/DownloadReportPage.unit.spec.jsx

```diff
@@ -418,3 +422,58 @@ describe('DownloadRecordsPage with last successful update timestamp', () => {
     expect(screen.getByTestId('new-records-last-updated')).to.exist;
   });
 });
+
+describe('DownloadRecordsPage - Oracle Health CCD not enabled', () => {
+  it('shows VistaOnlyContent for VistA-only users when flag is false', () => {
+    const state = getBaseState(vistaOnlyFacilities, []);
```

Comment:
NITPICK: We could probably enhance `getBaseState`  to accept a option for feature toggles